src/EPR.Payment.Facade/Validations/RegistrationFees/ComplianceScheme/ComplianceSchemeFeesRequestDtoValidator.cs

```diff
@@ -16,6 +16,10 @@ public ComplianceSchemeFeesRequestDtoValidator()
             RuleFor(x => x.ApplicationReferenceNumber)
                     .NotEmpty().WithMessage(ValidationMessages.ApplicationReferenceNumberRequired);
 
+            RuleFor(x => x.SubmissionDate)
```

Comment:
what if the user sends a future date? will it still pass this validation?

Reply:
added validation for future dates